go.mod

```diff
@@ -33,17 +35,17 @@ require (
 	github.com/pion/webrtc/v3 v3.1.5
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v1.11.0
-	github.com/russross/blackfriday/v2 v2.1.0 // indirect
+	github.com/rs/zerolog v1.25.0
 	github.com/stretchr/testify v1.7.0
 	github.com/thoas/go-funk v0.8.0
 	github.com/twitchtv/twirp v8.1.0+incompatible
 	github.com/urfave/cli/v2 v2.3.0
 	github.com/urfave/negroni v1.0.0
 	go.uber.org/zap v1.19.1
-	golang.org/x/mod v0.5.1 // indirect
-	golang.org/x/sys v0.0.0-20211007075335-d3039528d8ac // indirect
+	google.golang.org/grpc v1.42.0
```

Comment:
I guess we will need this when we do analytics server. But, if this is getting pulled in because of `ion-sfu`, it should not be necessary, i. e. we should not move over that code from `ion-sfu`.

Reply:
reference by session.go